src/api/app/controllers/webui/repositories_controller.rb

```diff
@@ -113,7 +113,14 @@ def destroy
   end
 
   # GET project/repository_state/:project/:repository
-  def state; end
+  def state
+    state_presenter = Repositories::StatePresenter.new(@project, @repository, @configuration)
+    render 'state', locals: {
```

Comment:
Why do you explicit call render here instead of passing on the presenter as instance variable?

Reply:
To avoid using instance variables, the locals provide a bit of checking for typos and everything else

Reply:
Hm, we don't avoid instance variables anywhere else. And I really think we should guard against typos by reviews and/or tests...

Reply:
Okay!